db/migrate/20200330194125_add_github_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGithubIdToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :github_id, :string
```

Comment:
Add github_id to users column as a way to find the user we want to add as a friend